tic1.js

```diff
@@ -0,0 +1,127 @@
+// this is a test
+const readline = require('readline');
+
+const readlineInterface = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+      readlineInterface.question(questionText, resolve);
+    });
+  }
+
+let cell1 = 1
+let cell2 = 2
+let cell3 = 3
+let cell4 = 4
+let cell5 = 5
+let cell6 = 6
+let cell7 = 7
+let cell8 = 8
+let cell9 = 9

```

Comment:
I think I see where you were going with this -- code using a symbol (`cell1`) can be clearer than using a bare number -- in that case using the keyword `const` instead of `let` will express that intention in the code above. 

However, it doesn't seem that you actually *use* these constants anywhere except in the initialization of the `cells` array, so my advice is to inline them by either using bare numbers, or a loop, to initialize `cells`.

Reply:
you asked me to clarify this in class -- "inline them by either using bare numbers, or a loop, to initialize cells":

bare numbers: `let cells = [1,2,3,4,5,6,7,8,9]`

a loop: 
```
let cells = [];
for (let i = 1; i<=9; ++i) {
  cells.push(i);
}
```